src/EnergyPlus/Coils/CoilCoolingDX.hh

```diff
@@ -142,8 +143,14 @@ struct CoilCoolingDX
     Real64 speedRatioReport = 0.0;
     Real64 wasteHeatEnergyRate = 0.0;
     Real64 wasteHeatEnergy = 0.0;
+    Real64 recoveredHeatEnergy = 0.0;
+    Real64 recoveredHeatEnergyRate = 0.0;
 };
 
+extern int const coilNormalMode; // Normal operation mode
+extern int const coilEnhancedMode; // Enhanced operation mode
+extern int const coilSubcoolReheatMode; // SubcoolReheat operation mode
```

Comment:
I guess these values are probably used by Unitary, and maybe in the unit test.  If they aren't then they should be taken out of the header.  But I bet they are, so nevermind.